views/components/progress_bar.erb

```diff
@@ -1,5 +1,5 @@
 <%# locals: (numerator:, denominator:, title:) %>
-<% progress = [100, 100 * numerator / denominator].min
+<% progress = [100, denominator.zero? ? 0 : 100 * numerator / denominator].min
```

Comment:
I feel like it would be better representation if we assume progress is 100% when the denominator is 0

Reply:
Makes sense. Changed it